src/js/math/collide.js

```diff
@@ -0,0 +1,40 @@
+export function convexPolygonsCollide(shape1, shape2) {
```

Comment:
rename this file into `CollisionHelper`, please and move it into `src/js/helpers`